Index: openjdk-jdk-10.0.1+10/src/hotspot/os/linux/os_linux.cpp
===================================================================
--- openjdk-jdk-10.0.1+10/src/hotspot/os/linux/os_linux.cpp	2011-12-20 19:15:05.000000000 +0100
+++ openjdk-jdk-10.0.1+10/src/hotspot/os/linux/os_linux.cpp	2011-12-20 19:18:22.411503442 +0100
@@ -2881,16 +2881,13 @@
 // If we are running with earlier version, which did not have symbol versions,
 // we should use the base version.
 void* os::Linux::libnuma_dlsym(void* handle, const char *name) {
-  void *f = dlvsym(handle, name, "libnuma_1.1");
-  if (f == NULL) {
-    f = dlsym(handle, name);
-  }
+  void *f = dlsym(handle, name);
   return f;
 }

 // Handle request to load libnuma symbol version 1.2 (API v2) only.
 // Return NULL if the symbol is not defined in this particular version.
 void* os::Linux::libnuma_v2_dlsym(void* handle, const char* name) {
-  return dlvsym(handle, name, "libnuma_1.2");
+  return NULL;
 }
