--- openjdk-jdk-10.0.1+10/src/jdk.jdwp.agent/share/native/libdt_socket/socketTransport.c.orig	2018-05-15 03:55:52.665274709 +0000
+++ openjdk-jdk-10.0.1+10/src/jdk.jdwp.agent/share/native/libdt_socket/socketTransport.c	2018-05-15 03:56:11.761362237 +0000
@@ -386,7 +386,7 @@
        return instr;
     }
 
-    *mask = htonl(-1 << (32 - m));
+    *mask = htonl((uint32_t)(~0) << (32 - m));
     return s;
 }
 
