--- openjdk-jdk-10.0.1+10/src/java.base/share/native/libfdlibm/k_rem_pio2.c	2018-03-08 14:25:34.959432079 +1100
+++ openjdk-jdk-10.0.1+10/src/java.base/share/native/libfdlibm/k_rem_pio2.c	2018-03-08 14:27:23.637385107 +1100
@@ -197,7 +197,8 @@
 
     /* compute q[0],q[1],...q[jk] */
         for (i=0;i<=jk;i++) {
-            for(j=0,fw=0.0;j<=jx;j++) fw += x[j]*f[jx+i-j]; q[i] = fw;
+            for(j=0,fw=0.0;j<=jx;j++) fw += x[j]*f[jx+i-j]; 
+	    q[i] = fw;
         }
 
         jz = jk;
