--- openjdk-jdk-10.0.1+10/src/java.base/share/native/libfdlibm/e_asin.c	2018-02-27 10:08:25.589121957 +1100
+++ openjdk-jdk-10.0.1+10/src/java.base/share/native/libfdlibm/e_asin.c	2018-02-27 10:14:25.333456152 +1100
@@ -99,10 +99,10 @@
                 if(huge+x>one) return x;/* return x with inexact if x!=0*/
             } else
                 t = x*x;
-                p = t*(pS0+t*(pS1+t*(pS2+t*(pS3+t*(pS4+t*pS5)))));
-                q = one+t*(qS1+t*(qS2+t*(qS3+t*qS4)));
-                w = p/q;
-                return x+x*w;
+            p = t*(pS0+t*(pS1+t*(pS2+t*(pS3+t*(pS4+t*pS5)))));
+            q = one+t*(qS1+t*(qS2+t*(qS3+t*qS4)));
+            w = p/q;
+            return x+x*w;
         }
         /* 1> |x|>= 0.5 */
         w = one-fabs(x);
