--- openjdk-jdk-10.0.1+10/src/java.base/unix/native/libjava/jdk_util_md.h.orig	2017-11-30 21:41:18.780720828 +1100
+++ openjdk-jdk-10.0.1+10/src/java.base/unix/native/libjava/jdk_util_md.h	2017-11-30 21:43:07.698579220 +1100
@@ -37,7 +37,7 @@
 #define ISNAND(d) isnan(d)
 #elif defined(__linux__) || defined(_ALLBSD_SOURCE)
 #include <math.h>
-#define ISNANF(f) isnanf(f)
+#define ISNANF(f) __isnanf(f)
 #define ISNAND(d) isnan(d)
 #elif defined(_AIX)
 #include <math.h>
